LeaderboardBackend/Controllers/LeaderboardsController.cs

```diff
@@ -1,51 +1,46 @@
-using Microsoft.AspNetCore.Mvc;
+using LeaderboardBackend.Controllers.Requests;
 using LeaderboardBackend.Models;
 using LeaderboardBackend.Services;
-using LeaderboardBackend.Controllers.Requests;
+using Microsoft.AspNetCore.Mvc;
 
-namespace LeaderboardBackend.Controllers
+namespace LeaderboardBackend.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class LeaderboardsController : ControllerBase
 {
-	[Route("api/[controller]")]
-	[ApiController]
-	public class LeaderboardsController : ControllerBase
+	private readonly ILeaderboardService _leaderboardService;
+
+	public LeaderboardsController(ILeaderboardService leaderboardService)
 	{
-		private readonly ILeaderboardService _leaderboardService;
-		public LeaderboardsController(
-			ILeaderboardService leaderboardService
-		)
-		{
-			_leaderboardService = leaderboardService;
-		}
+		_leaderboardService = leaderboardService;
+	}
 
-		[HttpGet("{id}")]
-		public async Task<ActionResult<Leaderboard>> GetLeaderboard(long id)
-		{
-			Leaderboard? leaderboard = await _leaderboardService.GetLeaderboard(id);
-			if (leaderboard == null)
-			{
-				return NotFound();
-			}
+	[HttpGet("{id}")]
+	public async Task<ActionResult<Leaderboard>> GetLeaderboard(ulong id)
+	{
+		return await _leaderboardService.GetLeaderboard(id) is Leaderboard leaderboard
+			? leaderboard
+			: NotFound();
```

Comment:
I personally find this less readable than the previous way it was written. What is the benefit of doing it this way?

Reply:
Addressed in 76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d (`LeaderboardsController.cs` [`R22-28`](https://github.com/leaderboardsgg/leaderboard-backend-poc/commit/76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d#diff-5e06add1f8b7159c467ddce62b6647055b4b6624a652526e6f7f41df6df94aebR22-R28)).